### ISSUE TITLE
cannot detach tmux session without closing/minimizing iTerm

### ISSUE THREAD

**Issue by pruett** [sorin-ionescu/prezto#1140]
I have the following set in `zpreztorc`

```
# Auto start a session when Zsh is launched in a local terminal.
zstyle ':prezto:module:tmux:auto-start' local 'yes'

# Auto start a session when Zsh is launched in a SSH connection.
zstyle ':prezto:module:tmux:auto-start' remote 'yes'
```

I cannot seem to detach <Prefix> + d out of the auto-start tmux session without closing iTerm. I am new to tmux, so I might be missing something obvious. Is this by design? Or is there a way to retain auto-start with the ability to detach?

**Comment by excetara2** [comment]
@pruett Yes I was wondering the same thing as well because I only have this set for use over ssh connections but if I detach then it disconnects. I couldn't figure out how to disable this setting so I could have more than one tmux session at once since it's not recommended to nest.
